### ISSUE TITLE
Allow to "pull all" from P8s kind of endpoints

### ISSUE THREAD

**Issue by pilhuhn** [hawkular/hawkular-openshift-agent#113]
Right now HOSA requires an explicit list of metrics to be collected.
To make startup more easy it should be possible that for P8s endpoints all the metrics, the endpoint delivers are forwarded to Hawkular-Metrics.

```
endpoints:
  - type: prometheus
    enabled: false
    url: http://127.0.0.1:8081/metrics
    collection_interval: 1m
    metrics:
    - name: hawkular_openshift_agent_metric_data_points_collected_total
    - name: go_goroutines
```

This request is to allow to skip the `metrics:` part and allow to just collect all the metrics found on the endpoint. Perhaps one could use a special name of `*` for this

```
endpoints:
  - type: prometheus
    metrics:
    - name: *
```

**Comment by jmazzitelli** [comment]
this should now be possible with the changes made in PR #117  - you simply do not specify "metrics:" which means you want them all.

**Comment by jmazzitelli** [comment]
@pilhuhn see if you can try PR #117  - I think that PR should also introduce this feature, too.

**Comment by jmazzitelli** [comment]
This feature is now in master branch. It is included in the last release 1.1.0.Final found on docker hub.
